src/minType.js

```diff
@@ -299,6 +299,22 @@ function minType (sup, sub) {
     return consistencyCheck(computed)
   }
 
+  if (subType === 'union' && superType !== 'union') {
+    let anyOf = sub.anyOf
+    let unionResult = sub
+    unionResult.anyOf = []
+    for (let i = 0; i < anyOf.length; i++) {
```

Comment:
Style-wise, perhaps this should be a for-of loop, as is done elsewhere in this function?